packages/editor/src/components/post-actions/index.js

```diff
@@ -0,0 +1,193 @@
+/**
+ * WordPress dependencies
+ */
+import { useSelect } from '@wordpress/data';
+import { store as coreStore } from '@wordpress/core-data';
+import { useMemo, useState, Fragment, Children } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import {
+	privateApis as componentsPrivateApis,
+	Button,
+	Modal,
+} from '@wordpress/components';
+import { moreVertical } from '@wordpress/icons';
+
+/**
+ * Internal dependencies
+ */
+import { unlock } from '../../lock-unlock';
+import { usePostActions } from './actions';
+import { store as editorStore } from '../../store';
+
+const {
+	DropdownMenuV2: DropdownMenu,
+	DropdownMenuGroupV2: DropdownMenuGroup,
+	DropdownMenuItemV2: DropdownMenuItem,
+	DropdownMenuItemLabelV2: DropdownMenuItemLabel,
+	DropdownMenuSeparatorV2: DropdownMenuSeparator,
+	kebabCase,
+} = unlock( componentsPrivateApis );
+
+const POST_ACTIONS_WHILE_EDITING = [
+	'view-post',
+	'view-post-revisions',
+	'rename-post',
```

Comment:
The `rename-post` action is probably redundant for standard post types. The title is available in Canvas, and if the post type doesn't support the `title` field, then renaming should be disabled in the editor.